### PR TITLE
Ext inn ar

### PR DESCRIPTION
**Pull request by romani79**

### CODE REVIEW

File: notebooks/external_innervation.ipynb [hunk 1]
```diff
@@ -185,6 +239,13 @@
     "offset = br_metadata[\"space origin\"]"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Collect positions of synapses and map it to the different layers"
```
Comment (alex4200) [inline notebooks/external_innervation.ipynb]:
```suggestion
    "Collect positions of synapses and map _them_ to the different layers"
```

File: notebooks/external_innervation.ipynb [hunk 2]
```diff
@@ -235,11 +302,12 @@
...
     "mod_y_distr = mod_y_distr/mod_y_distr.sum()\n",
-    "exp_y_distr = [0.2476014415, 0.0705293209, 0.6792951748, 0.0025740628]\n",
+    "exp_y_distr = [0.2476014415, 0.0705293209, 0.6792951748, 0.0025740628] # Bezaire and Soltesz, 2013\n",
     "x = np.arange(4)\n",
+    "width = 0.35\n",
     "fig, ax = plt.subplots()\n",
-    "ax.bar(x, mod_y_distr, label='simulation')\n",
-    "ax.plot(x,exp_y_distr,color='r', label='expected')\n",
+    "ax.bar(x-width/2, mod_y_distr, width, label='simulation')\n",
+    "ax.bar(x+width/2, exp_y_distr, width, color='r', label='expected')\n",
```
Comment (alex4200) [inline notebooks/external_innervation.ipynb:310]: Maybe the plot is a bit more clear when the y axis is log-scale? 
ax.semilogy()

File: notebooks/external_innervation.ipynb [hunk 3]
```diff
@@ -300,6 +394,38 @@
...
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Excercise\n",
+    "Calculate how many SC synapses a pyramidal cell or an interneuron receive on average. Plot the distribution and provide mean and std. Put a tuple of mean and std in 'ans_1a' (pyramidal cell) and 'ans_1b' (interneuron) respectively.\n",
+    "\n",
+    "The pandas dataframe syns contains only gid of the postsynaptic cells but not the mtype. For this information, you should need to load the circuit and create a Circuit object (see 2.5)"
```
Comment (alex4200) [inline notebooks/external_innervation.ipynb]: What do you mean with "see 2.5"? I don't get it.

File: notebooks/external_innervation.ipynb [hunk 4]
```diff
@@ -300,6 +394,38 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
```
Comment (alex4200) [inline notebooks/external_innervation.ipynb:415]: I would insert a cell just before that and indicate that this is a cell where the user can work the exercise.

File: notebooks/external_innervation.ipynb [hunk 5]
```diff
@@ -300,6 +394,38 @@
...
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# This is to generate the answers to paste in the submission box below.\n",
+    "# Run this and copy-paste the output into the box below\n",
```
Comment (alex4200) [inline notebooks/external_innervation.ipynb]:
```suggestion
    "# After you defined the variables with your answers, run this cell and the next cell, and copy-paste the output into the box below\n",
```

**Review by alex4200 — CHANGES_REQUESTED** [review]
Much better with the comments now, and just a few suggestions for the notebook.

File: notebooks/external_innervation.ipynb (hunk 2 above)
Comment (romani79) [reply notebooks/external_innervation.ipynb:310]: I would prefer to keep linear axis. I prefer to stress the fact that most of the synapses are in SR and SO.

Comment (alex4200) [reply notebooks/external_innervation.ipynb:310]: Sure, makes sense

**Comment by alex4200** [comment]
Thanks, looks good to me now!

**Pull request merged**